keywhizfs/main.go

```diff
@@ -108,6 +115,18 @@ func main() {
 		server.Unmount()
 	}()
 
+	// Setup metrics
+	hostname, err := os.Hostname()
+	if err != nil {
+		log.Fatalf("unable to get hostname: %s", err)
+		os.Exit(1)
+	}
+	metrics := keywhizfs.NewMetricsConfig(*metricsURL, fmt.Sprintf("go-kwfs.%s", mountpoint), hostname, logConfig)
```

Comment:
I wanted a unique string to make it easier to find the metrics, wasn't sure if anything else was logging under keywhizfs. Will change before merge.